yarn.lock

```diff
@@ -4346,6 +4382,12 @@ katex@^0.8.3:
   dependencies:
     match-at "^0.1.0"
 
+katex@^0.9.0-alpha1:
```

Comment:
It seems to me that the patch adds a bunch of duplicate dependencies can they be cleaned out or placed in a separate patch?